lakers-python/src/lib.rs

```diff
@@ -67,13 +66,26 @@ impl AutoCredential {
     }
 }
 
-// this name must match `lib.name` in `Cargo.toml`
+/// Lakers implementation of EDHOC.
+///
+/// The `EdhocInitiator` and `EdhocResponder` are entry points to this module.
+///
+/// Operations in this module produce logging entries on the `lakers.initiator` and
+/// `lakers.responder` logger names. Due to implementation details of `pyo3_log`, Python's log
+/// levels are cached in the Rust implementation. It is recommended that the full logging
+/// is configured before creating Lakers objects. A setup with `logging.basicConfig(loglevel=5)`
```

Comment:
Maybe best to add log level name? like `logging.basicConfig(level=logging.DEBUG)`

Reply:
Actually I am still confused on how to use it. 

I tried:

```python
import lakers, logging
logging.basicConfig(level=logging.DEBUG)
lakers.EdhocInitiator()
```

but after `python3 testlog.py` nothing got emitted.

So maybe I misunderstood the part about the hardcoded logs from Rust?

Reply:
I am also not sure it is a good idea to have Rust always-logging by default. I guess I could live with it sending only trace logs.
On the other hand, if we add more logs to the Rust side, we should make sure the python wrapper sets the Rust level to trace during build.

Reply:
So, the thing is that Python has no level below DEBUG (which Rust's log knows as 'trace'). That's why the docs say `basicConfig(level=5)`, which is even below DEBUG.

This also places it in a convenient spot where just enabling Python logging doesn't come on unintentionally, and pyo3_log's caching means that there should not be a practically relevant performance impact.

Reply:
Thank you, I just tested and it works as you said.

(previously I had misunderstood, and thought that traces logs would _always_ show even when python logs are off, but I verified that this is not the case)